src/rust/ide/src/double_representation/module.rs

```diff
@@ -416,6 +416,16 @@ impl Info {
         index_to_place_at
     }
 
+    /// Add a new import if the module is not already imported.
+    pub fn import_module_if_not_already_imported
```

Comment:
I would name it shorter `get_or_import_module` or simply `get_module`

Reply:
But you're not getting module, only the index of the import. I may rename it to `add_module_import`

Reply:
Would it make sense to have a newtype for the `usize` here? `ModuleIndex` or something like it?

Maybe then call it `import_module` and describe the behaviour for already imported modules in the doc string.

What does it mean if this returns `None`? Is this an error?

Reply:
Actually this is index of the module's line. Could we make a newtype for it? Perhaps we could, however it again involves refactoring in many places (so far we used usize for module's line index). And I don't want to make this PR any longer.

None means that the module wasn't added. It could contain the line with current import. However, I think I will remove the index from the return type, as it isn't used anywhere.